examples/timeseries/timeseries_anomaly_detection.py

```diff
@@ -102,25 +87,12 @@ def plot_dates_values(data):
 """
 
 
-def get_value_from_df(df):
-    return df.value.to_list()
-
-
-def normalize(values):
-    mean = np.mean(values)
-    values -= mean
-    std = np.std(values)
-    values /= std
-    return values, mean, std
-
-
-# Get the `value` column from the training dataframe.
-training_value = get_value_from_df(df_small_noise)
-
-# Normalize `value` and save the mean and std we get,
+# Normalize and save the mean and std we get,
 # for normalizing test data.
-training_value, training_mean, training_std = normalize(training_value)
-len(training_value)
+training_mean = df_small_noise.mean()
+training_std = df_small_noise.std()
+df_training_value = (df_small_noise - training_mean) / training_std
+len(df_training_value)
```

Comment:
While we're at it -- please print a complete sentence here, e.g. "Number of training samples: %d"